### PR TITLE
Polish README

### PR DESCRIPTION
**Pull request by jswrenn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,31 +3,82 @@
...
+            tokio::time::sleep(frequency).await;
+        }
+    });
+}
 
-    // Wait some time before sampling metrics again.
-    tokio::time::sleep(Duration::from_millis(100)).await;
+// instrument some tasks and await them
+loop {
+    monitor.instrument(do_work()).await;
```
Comment (carllerche) [inline README.md:36]: I would always show this inside of a `tokio::spawn(monitor.instrument(....))` to hammer that pattern.

File: README.md [hunk 2]
```diff
@@ -3,31 +3,82 @@
...
+environment variable before compiling your application; e.g.:
+```sh
+RUSTFLAGS="--cfg tokio_unstable" cargo build
+```
+Or, by creating the file `.cargo/config` in the root directory of your crate
+with the contents:
+```toml
+[build]
+rustflags = ["--cfg", "tokio_unstable"]
+rustdocflags = ["--cfg", "tokio_unstable"]
```
Comment (carllerche) [inline README.md:54]: This isn't needed for the user.

File: README.md [hunk 3]
```diff
@@ -3,31 +3,82 @@
...
+let frequency = std::time::Duration::from_millis(500);
+tokio::spawn(async move {
+    for metrics in runtime_monitor.intervals() {
+        println!("Metrics = {:?}", metrics);
+        tokio::time::sleep(frequency).await;
+    }
+});
+
+// run some tasks
+tokio::join![do_work(), do_work(), do_work(),];
```
Comment (carllerche) [inline README.md:79]: spawns will be better here.

File: README.md (hunk 2 above)
Comment (jswrenn) [reply README.md:54]: Having `rustdocflags` is helpful if you `cargo doc`, right?

**Review by carllerche — APPROVED** [review]

**Pull request merged**